taqueria-plugin-ligo/_readme.eta

```diff
@@ -60,6 +60,16 @@ Lastly, `taq compile hello.mligo` will compile `hello.mligo` and emit `hello.tz`
 
 The `--json` flag will make the task emit JSON-encoded Michelson instead of pure Michelson `.tz`
 
+## The `taq compile-all` Task
+
+Basic usage is:
+
+```shell
+taq compile-all
```

Comment:
What is an "eta" file and why does it repeat the same information as the "md"?

Reply:
Not sure, I am inheriting the design from others. It's basically the same as README but I was told that "eta" file is to be updated